Sources/Fuzzilli/Core/Corpus.swift

```diff
@@ -84,13 +86,24 @@ public class Corpus: ComponentBase {
         if program.size > 0 {
             programs.append(program)
             ages.append(0)
+            cacheTypeExtensions(program: program)
         }
     }
     
     /// Adds multiple programs to the corpus.
     public func add(_ programs: [Program]) {
         programs.forEach(add)
     }
+
+    /// Change type extensions for cached ones to save memory
+    private func cacheTypeExtensions(program: Program) {
+        var cachedRuntimeTypes = VariableMap<Type>()
+        for (variable, runtimeType) in program.runtimeTypes {
+            let cachedType = runtimeType.getCachedType(cache: &typeExtensionDeduplicationSet)
```

Comment:
Mostly just a semantic difference, but I think `uniquify` is the better term for what this achieves, since the goal is not to improve the speed of future lookups or anything like that (as a cache would) but to reduce memory usage by uniquifying instances